docs/README.md

```diff
@@ -57,15 +57,18 @@ account = client.get_account("code-benjamin.dumonde@example.com")
 
 ### Pagination
 
-There are 2 methods for pagination:
+There are 4 methods for pagination:
```

Comment:
What do you think about separating the first/count methods into their own subsection kind of like we do here: https://recurly.github.io/recurly-client-node/#efficiently-fetch-the-first-or-last-resource

I don't really feel like it's accurate to call them a "method of pagination" since they don't paginate. They are "methods on the pager" so i can see how it could be construed that way.

Reply:
Updated